app/assets/javascripts/pension-pot.js

```diff
@@ -85,7 +85,10 @@
           that.$appointmentCountWrapper.fadeOut();
         } else {
           that.$appointmentCountWrapper.fadeIn().attr('aria-live', 'polite');
-          that.$onePreviousAppointmentInput.prop('checked', true);
+          // do not reset the value to 1 when the page is reloaded
```

Comment:
Maybe just add that this is for the case when the form is submitted but re-rendered with errors and ensures the state of these radio buttons is maintained.